code/Form/UserForm.php

```diff
@@ -163,7 +162,7 @@ public function getFormActions()
     /**
      * Get the required form fields for this form.
      *
-     * @return RequiredFields
+     * @return UserFormsRequiredFields
```

Comment:
This is technically an API change. If someone had extended `getRequiredFields`, it won't return a `UserFormsRequiredFields`.

Return a `UserFormsRequiredFields` is fine because it extends `RequiredFields`, but the PHPDoc should still state that it returns a `RequiredFields`